### DISCUSSION TITLE
battery aging cost question

### DISCUSSION THREAD

**Discussion by woojmn** [DLR-VSDC/IEEE-MVC-2023#32] [Q&A]
Hello,

I have a question about battery aging cost.

The battery aging cost is shown as follow picture at the cycle #4.
![J_batAging picture](https://user-images.githubusercontent.com/95162423/216521577-1c15a8ca-174b-4443-9a5c-68437f6d655d.PNG)

In the picture, there is a section where the battery aging cost decreases while driving. Is it possible?

I think that it require to be integrated.

_Originally posted by @woojmn in https://github.com/DLR-VSDC/IEEE-MVC-2023/discussions/26_

**Comment by JBRDLR** [comment]
Dear @woojmn, the battery aging cost uses the average values e.g. cell current (for reason of model simplification) of the whole simulation run, so only the last value of the cost function is relevant and explains the behavior shown above.
Best Jonathan

**Comment by woojmn** [comment]
Then, if the driving ends in 200 seconds, is the battery aging cost larger than if the driving ends in 1000 seconds?

As I understand, a larger battery aging cost reduces battery lifespan more.
How can battery lifespan increase with more driving?

It is weird. The battery lifespan should decrease as the battery is used. 
In addition, the already reduced battery life cannot be recovered while driving.

**Comment by JBRDLR** [comment]
Please refer to the paper:
![image](https://user-images.githubusercontent.com/110536277/217554312-64d45c65-871f-45ce-baca-f86d8fded905.png)

@rpintodecastro probably you can give a better answer?

**Comment by rpintodecastro** [comment]
Dear @woojmn

This battery aging indicator takes into account the average current and temperatures of the battery over the cycle: 
batAging = f(I_avg, T_avg, ...)
In the simulation results you shared, I think the average current is significantly higher during the initial 200s, which leads to an increase in this indicator. 

Note also that the units of the battery aging indicator are in "mA/cycle". This means that this indicator capture the rate  of variation of battery capacity (and not the absolute capacity loss over the simulation, as indicated in your comments).  
Imagine that, at the end of the simulation (200s), you obtain a value of 100 mA/cycle with average current 0.7C and temperature 30deg. This means that, if you keep using the battery with the same average current and temperature until the end of the discharge cycle, your battery would lose 100mA of capacity.
